examples/td3/config.yaml

```diff
@@ -1,21 +1,15 @@
 # Environment
 env_name: HalfCheetah-v4
-env_task: ""
 exp_name: "debugging"
-env_library: gym
-record_video: 0
-normalize_rewards_online: 0
-normalize_rewards_online_scale: 5
-normalize_rewards_online_decay: 0.99
 total_frames: 1000000
-frames_per_batch: 1000
```

Comment:
Not a big issue but I might put this back to 1000 as its a common length for MuJoCo envs

Reply:
not sure
this is the frames in each batch but not the truncation length.
This should be handled by `max_frames_per_traj`

Reply:
Oh, yes! I was thinking of env like halfcheetah. But you are right for Hopper or others where episodes can be less than 1000 so multiple epochs per batch!